<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pump11EliteWindow</class>
 <widget class="QDialog" name="Pump11EliteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pump 11 Elite</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>355</width>
     <height>104</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_volume">
    <item row="1" column="3">
     <widget class="QToolButton" name="button_read_volume_infuse">
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QToolButton" name="button_clear_volume_infuse">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QToolButton" name="button_clear_volume_target">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QToolButton" name="button_read_volume_withdraw">
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QToolButton" name="button_clear_volume_withdraw">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_volume_target">
      <property name="text">
       <string>Target volume (µL):</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_volume_target">
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_volume_infuse">
      <property name="text">
       <string>Infused (µL):</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_volume_infuse">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_volume_withdrawn">
      <property name="text">
       <string>Withdrawn (µL):</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_volume_withdraw">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolButton" name="button_read_all">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Read All</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>211</width>
     <height>84</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_rate">
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_rate_infuse">
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_rate_withdraw">
      <property name="text">
       <string>W Rate (µL/min):</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_rate_withdraw">
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_rate_infuse">
      <property name="text">
       <string>I Rate(µL/min):</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>324</width>
     <height>104</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_move">
    <item row="1" column="0">
     <widget class="QToolButton" name="button_infuse">
      <property name="text">
       <string>Infuse</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="button_run">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="button_stop">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="button_withdraw">
      <property name="text">
       <string>Withdraw</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QToolButton" name="button_reverse">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Reverse</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="button_reset_to_start">
      <property name="text">
       <string>Reset to start position</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
